src/coveralls.jl

```diff
@@ -78,18 +78,11 @@ module Coveralls
             error("No compatible CI platform detected")
         end
 
-        repo_token =
-                get(ENV, "COVERALLS_TOKEN") do
-                    get(ENV, "REPO_TOKEN") do #backward compatibility
-                        if data["service_name"] != "travis-ci"
-                            error("Coveralls submission requires a COVERALLS_TOKEN environment variable")
-                        end
-                    end
-                end
-        if repo_token != nothing
-            data["repo_token"] = repo_token
-        end
+        data = add_repo_token(data, false)
 
+<<<<<<< HEAD
```

Comment:
You have some merge conflict markers committed here.

Reply:
Basically, you may want to drop that last commit, and instead rebase your PR onto the latest master.

While at it, you have a commit "Fixed faulty previous commit, factored out add_repo_token" -- perhaps just squash that into the preceding commit (using an interactive rebase, `git rebase -i`, quite handy tool); similarly, the commit "Fixed bug in call to add_repo_token" could be squashed into the second commit.

If you don't know how to do any of these, but are interested in learning, I'd be happy to coach you through it, too. It really isn't hard.

Reply:
I wasn't sure whether you'd want me to squash, it can be confusing when reviewing. I have now squashed all commits together.